File: 1528-kids-with-the-greatest-number-of-candies/1528-kids-with-the-greatest-number-of-candies.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
        maxCandies = max(candies[:])
        boolList = []

        for i in range(len(candies)):
            if ((candies[i] + extraCandies) >= maxCandies):
                boolList.append(True)
            else: 
                boolList.append(False)
        return boolList
             
        

File: 1528-kids-with-the-greatest-number-of-candies/1528-kids-with-the-greatest-number-of-candies.cpp
class Solution {
public:
    vector<bool> kidsWithCandies(vector<int>& candies, int extraCandies) {
        int maxCandies = *max_element(candies.begin(), candies.end()); 
        vector<bool> boolList;

        for (int i = 0; i < candies.size(); i++){
            if (candies[i] + extraCandies >= maxCandies){
                boolList.push_back(true);
            }
            else{
                boolList.push_back(false);
            }
        }
        return boolList;

    }
};
